XMonad/Layout/IndependentScreens.hs

```diff
@@ -122,6 +147,48 @@ onCurrentScreen f vws = screen . current >>= f . flip marshall vws
 countScreens :: (MonadIO m, Integral i) => m i
 countScreens = fmap genericLength . liftIO $ openDisplay "" >>= liftA2 (<*) getScreenInfo closeDisplay
 
+-- | Replacement for 'copy' from "XMonad.Actions.CopyWindow"
+-- Copy the focused window to a workspace on the current screen.
+-- Key binding example:
+--
+-- > ((modm, xK_v ), copyWindowTo 1) -- @@ copy focused window to the 1st workspace on current screen.
+--
+copyWindowTo :: Int->X ()
+copyWindowTo j = do
+             sort <- getSortByIndex
+             ws <- gets (sort . W.workspaces . windowset)
+             sc <- gets (W.screen . W.current . windowset)
+             let wsID = [ W.tag m | m<-ws, wsConvertToNum m == sc]
+             windows $ copy $ wsID !! (j-1)
+
+-- Take a 'ScreenId' and return a String.
+-- Using @show@ will get "S 0" for screen 0.
+showScreenId :: ScreenId->String
+showScreenId sc = [ last (show sc) ]
```

Comment:
This is just wrong, drop this please. If anyone needs the number, `fromIntegral` works well.